modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/portlet/action/EditObjectEntryRelatedModelMVCActionCommand.java

```diff
@@ -71,12 +71,11 @@ private void _addObjectRelationshipMappingTableValues(
 				_objectDefinitionLocalService.getObjectDefinition(
 					objectRelationship.getObjectDefinitionId2());
 
-			_objectRelationshipLocalService.
-				addObjectRelationshipMappingTableValues(
-					_portal.getUserId(actionRequest), objectRelationshipId,
-					objectEntryId, objectRelationshipPrimaryKey2,
-					ServiceContextFactory.getInstance(
-						objectDefinition.getClassName(), actionRequest));
+			_objectRelationshipService.addObjectRelationshipMappingTableValues(
+				objectRelationshipId, objectEntryId,
+				objectRelationshipPrimaryKey2,
+				ServiceContextFactory.getInstance(
+					objectDefinition.getClassName(), actionRequest));
```

Comment:
`_objectRelationshipService.addObjectRelationshipMappingTableValues()` > Here, you check the update permission.

Before calling this method, the method `getObjectDefinition()` is called by the `LocalService` class. Because of that, I would like to know where the view permission is being checked in this specific user case.

Reply:
@marcelabc here they are, [see](https://github.com/liferay/liferay-portal/blob/32feea6d8af590540fcd358d25a5a3b37c3b471b/modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipServiceImpl.java#L81) and [see](https://github.com/liferay/liferay-portal/blob/32feea6d8af590540fcd358d25a5a3b37c3b471b/modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipServiceImpl.java#L93)

Reply:
Okay, thank you for pointing out :+1: